code/modules/mob/living/carbon/human/species_types/golems.dm

```diff
@@ -1089,30 +1089,33 @@
 	for(var/obj/effect/proc_holder/spell/aoe_turf/knock/spell in C.mob_spell_list)
 		C.RemoveSpell(spell)
 
-/datum/species/golem/capitalist/spec_unarmedattacked(mob/living/carbon/human/user, mob/living/carbon/human/target)
+/datum/species/golem/capitalist/proc/handle_speech(datum/source, list/speech_args)
+	playsound(source, 'sound/misc/mymoney.ogg', 25, FALSE)
+	speech_args[SPEECH_MESSAGE] = "Hello, I like money!"
+
+/datum/species/golem/capitalist/spread
+	info_text = "As a <span class='danger'>Capitalist Golem</span>, your fist spreads the powerful industrializing light of capitalism."
```

Comment:
could you maybe give them a special name as well or something, just so that people can more easily realize that "hey, I can convert people by punching them!" (outside of a literal three word change in only their greeting text)

like, I dunno, call them "Lesser Capitalist" or "Greater Capitalist" golems (respectively) or something

Reply:
I'll come up with some kind of flavor name for them. I could call the spreading Soviet golem something like 'Tzar Golem'.